server/protocols/jmap-rfc-8621/src/test/scala/org/apache/james/jmap/routes/JMAPApiRoutesTest.scala

```diff
@@ -303,7 +331,47 @@ class JMAPApiRoutesTest extends AnyFlatSpec with BeforeAndAfter with Matchers {
         .body(WRONG_OBJECT_REQUEST)
       .when()
         .post
-      .then
+      .`then`
+        .statusCode(HttpStatus.SC_BAD_REQUEST)
+        .body("status", equalTo(400))
+        .body("type", equalTo(RequestLevelErrorType.NOT_REQUEST.value))
+        .body("detail", equalTo("The request parsed as JSON but did not match the type signature of the Request object: Invalid RequestObject: List((/methodCalls,List(JsonValidationError(List(error.expected.jsarray),ArraySeq()))))"))
```

Comment:
the error message seems to be a bad serialization, it is not enough user friendly

Reply:
I personally think the message is pretty clear? The request does not match RequestObject structure and the last part tells you it's something related to methodCalls. I don't know how to be more specific in this case.

Reply:
what does mean `List(JsonValidationError(List(error.expected.jsarray)`. Isn't there a more friendly error message available?